MC/CustomGenerators/PWGDQ/Hijing_Jpsiee002.C

```diff
@@ -3,8 +3,9 @@ GeneratorCustom(TString ptParamPrompt="UserParam_PbPb5TeV_0010")
 {
   TString simulation = gSystem->Getenv("CONFIG_SIMULATION");
     
-  AliGenCocktail *ctl = GeneratorCocktail("Hijing_Jpsiee002");
-
+  AliGenerator   *gen =  GeneratorCocktail("Hijing_Jpsiee002");
```

Comment:
Same as above

Reply:
I thought since the macro returns AliGenerator it should return this. I tested and indeed it works just casting.